src/main/java/jmgomez/apipolicy/controller/PolicyController.java

```diff
@@ -20,18 +20,18 @@ public PolicyDto[] getPolicies() {
     }
 
     @GetMapping("/policies/{id}")
-    public PolicyDtoCov getPolicyByIDs(@PathVariable String id) {
-        return policyService.getPoliciesByIDs(getUserId());
+    public PolicyDtoCov getPolicyByIDs(@PathVariable("id") String id) {
+        return policyService.getPoliciesByIDs(getUserId() + id);
     }
 
     @GetMapping("/policies/{id}/accidents")
-    public Accident[] getAccidentsByPolicies(@PathVariable String id){
-        return policyService.getAccidentsByPolicies(getUserId());
+    public Accident[] getAccidentsByPolicies(@PathVariable("id") String id){
+        return policyService.getAccidentsByPolicies(getUserId() + id);
```

Comment:
Same problem than the previous one, why are you using the userID? this doesn't even make sense based on mock data, it won't work. I can't understand your logic here.